### PR TITLE
Removes a chunk of the readme, touches up other parts

### PR DESCRIPTION
**Pull request by iandonovan**
### Why?

- The README had a lot of old information (rot) and cruft.

### What?

- Adds a notice about Tape 1.7 no longer supporting Ubuntu 14
- Updates the readme to reference Ubuntu 16 throughout
- Removes local system requirements
- Removes all of the tutorial content, since:
  - We link to a tutorial article
  - It's not our responsibility to document how Digital Ocean works

### CODE REVIEW

**Comment by dkniffin** [comment]
It'd be nice to also have a table of contents for these docs. There's an app for that: http://doctoc.herokuapp.com/

File: README.md [hunk 1]
```diff
@@ -166,95 +152,23 @@ gem 'pg'
...
-```
-tape ansible deploy
-```
-
-This will git pull the latest changes from the tracking branch you specified and restart all services via monit.
 
-This command runs all Ansible roles specified in the deploy.yml playbook.
+To continue the deploy, SSH into the server as the `deployer` user and create your `config/secrets.yml` file in the app directory. The deploy will automatically continue when the file is saved.
 
 ### Rake tasks
```
Comment (dkniffin) [inline README.md]: I think this functionality is actually not working anymore. I broke it when I rewrote the run layer. We might want to delete this section and create a new issue for fixing that.

File: README.md [hunk 2]
```diff
@@ -101,7 +103,6 @@ tape ansible deploy -l staging
 ## Testing
```
Comment (dkniffin) [inline README.md]: I wonder if this whole testing and development section could be moved to a CONTRIBUTING.md

**Comment by dkniffin** [comment]
A few small things, but otherwise, LGTM. 👍

File: README.md (hunk 2 above)
Comment (BrandonMathis) [inline README.md]: Contributing.md is typically reserved for guidelines on how to submit PRs and Issues.

File: README.md (hunk 1 above)
Comment (BrandonMathis) [inline README.md]: I say leave it in here for now and lets get this functionality fixed :)

**Pull request merged**